lookups/v1/phone_number/fetch-lookup-get-cname-example-1/fetch-lookup-get-cname-example-1.3.x.js

```diff
@@ -6,5 +6,5 @@ const client = require('twilio')(accountSid, authToken);
 
 client.lookups.phoneNumbers('+16502530000')
               .fetch({type: 'caller-name'})
-              .then(phone_number => console.log(phone_number.callerName))
+              .then(phone_number => console.log(phone_number.carrier))
```

Comment:
Probably will be fixed by https://code.hq.twilio.com/twilio/api-definitions/pull/911#pullrequestreview-185355, but this instance should stay as callerName